src/cli/commands/dev.ts

```diff
@@ -89,36 +98,16 @@ export class Dev implements Command {
           }
 
           runDebouncedReflection(layout)
+
+          return {
+            entrypointScript: getTranspiledStartModule(layout, reflectionMode),
```

Comment:
So now before every restart the hook allows altering what entrypoint source is? Cool

Reply:
Yeah. Makes me think it's dangerous to expose that on the plugin interface. Here, what this enables is the ability to refresh the required schema modules in the start module whenever a file changes.

This used to not be required because we were looking for schema modules based on the filenames. So recomputing the layout whenever a file was added/deleted was enough.

But now that we find schema modules based on their import to Nexus, we need to recompute that on every file change. Here's a simple use-case where this is needed:

1. You create a file (that file is still not considered a nexus module)
2. You add `import { schema } from 'nexus'`
3. The start module needs to be updated to contain a `require` to that file
4. You comment out the `import` to nexus from that file
5. The start module needs to be updated to no longer contain a `require` to that file

See, nexus modules are no longer tight to file creation/deletion anymore